# RGB camera image topic, default is for openni2
camera_topic: "/cv_camera/image_raw"
#camera_topic: "/camera/rgb/image_rect_color"

# Depth image topic, default is for openni2
depth_topic: "/camera/depth_registered/hw_registered/image_rect"

# Optional video file path for reading from file.  if "no", read from ROS message
#video_file_path: "no"

show_cv_debug_window: True
